test/catalog.tests.bom

```diff
@@ -11,6 +11,16 @@ brooklyn.catalog:
       - type: cassandra-cluster-template
```

Comment:
V minor / ignore can you give the catalog item a name so that it doesnt show up as `Application` when ran